server/controllers/request.js

```diff
@@ -0,0 +1,77 @@
+// require Request Model
+const Request = {};
+
+exports.getRequestById = (req, res, next, id) => {

```

Comment:
Should we add some validation to this controller to see if the logged in user is a part of this request?

Reply:
yes, we can add `protect` middleware to for user validation.

Reply:
I think the point was that if we fetch a request by Id, we should make sure it belongs to the user requesting it